studiopress/genesis-custom-blocks
# Display the help text in the TextareaArray setting

(Testing instructions still valid as of May 5th)

## Before
<img width="554" alt="before-no-help" src="https://user-images.githubusercontent.com/4063887/106958725-5b562d00-66ff-11eb-927b-c13ef5439dcc.png">


## After

<img width="555" alt="multiselect-field" src="https://user-images.githubusercontent.com/4063887/106958740-5f824a80-66ff-11eb-8fb7-c60944495ac9.png">

#### Changes
* Adds help text to `<textarea>` settings in Multiselect, Select, and Radio fields 

#### Testing instructions
1. `composer install && npm install && npm run dev`
2. `/wp-admin` > Custom Blocks > Add New
3. Add a new field
4. Change the Field Type to Select
5. Expected: the new help text displays: <img width="572" alt="help-text" src="https://user-images.githubusercontent.com/4063887/106959895-f26fb480-6700-11eb-8e9d-da10b1ae66ed.png">
6. Change the Field Type to Multiselect
7. Expected: there's new help text under 2 settings: <img width="555" alt="multiselect-field" src="https://user-images.githubusercontent.com/4063887/106959973-103d1980-6701-11eb-93f8-49079f90909d.png">
8. Change the Field Type to Radio
9. Expected: there's new help text:
 <img width="549" alt="radio-new-help-text" src="https://user-images.githubusercontent.com/4063887/106960123-3a8ed700-6701-11eb-8669-8751abfb340f.png">
